examples/marketing-model/ml_marketing_banking_model.ipynb

```diff
@@ -0,0 +1,1902 @@
+      ]
+    },
+    {
+      "metadata": {
+        "id": "S0VuViuwL5rU",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "!gcloud services enable ml.googleapis.com\n",
+        "!gcloud services enable bigquery-json.googleapis.com"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "DaBNb7V0SAUq"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The data that we will be using for this demo is the [UCI Bank Marketing Dataset](https://archive.ics.uci.edu/ml/datasets/bank+marketing). \n",
+        "\n",
+        "The first step we will need to take is to create a BigQuery dataset and a table so that we can store this data. Make sure that you replace `yourdataset` and `yourtable` variables with any dataset and table name you want."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "ghr_crNYXH9w",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "your_dataset = 'your_dataset'\n",
+        "your_table = 'your_table'\n",
+        "project_id = os.environ[\"GOOGLE_CLOUD_PROJECT\"]"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "OQFkRqGrDL0V",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "!bq --location=US mk -d {project_id}:{your_dataset}\n",
+        "!bq mk -t {your_dataset}.{your_table}"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "dyI-AWNsqnJQ",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "There is a public dataset avaliable which has cleaned up some of the rows in the `UCI Bank Marketing Dataset`. We will download this file in the next cell and save locally as `data.csv`."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "r950lGFCcvQd",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "!curl https://storage.googleapis.com/erwinh-public-data/bankingdata/bankingdata --output data.csv"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "cHvOu8R7rzRL",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "We will now upload the `data.csv` file into our BigQuery table."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "2ltxd27fZeC3",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "!bq --location=US load --source_format=CSV {your_dataset}.{your_table} data.csv"
```

Comment:
Why provide the location param here? Will this fail if someone places their dataset in another region?

Reply:
Agreed, did not think about this